tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Certificates/CustomIdentityServerBuilderExtensionTests.cs

```diff
@@ -17,9 +16,22 @@
 namespace NHSD.BuyingCatalogue.Identity.Api.UnitTests.Certificates
 {
     [TestFixture]
-    [Parallelizable(ParallelScope.All)]
+    [Parallelizable(ParallelScope.None)]
     internal sealed class CustomIdentityServerBuilderExtensionTests
     {
+        private static void SetupEmbeddedCertificateFile()
+        {
+            using var stream = Assembly.GetExecutingAssembly()
+                .GetManifestResourceStream(typeof(CustomIdentityServerBuilderExtensionTests).Namespace + ".certificate.pfx").ThrowIfNull();
+            using var outputStream = File.Create("certificate.pfx");
+            stream.CopyTo(outputStream);
+        }
+
+        private static void RemoveCertificateFile()
+        {
+            File.Delete("certificate.pfx");
+        }
+
```

Comment:
I think the order of the methods here is a bit odd – I would normally expect to see the `public` methods before `private` ones (but within that order `static` before instance). For example:

```c#
public async Task UseValidPath_Sets_Certificate()
{
}

private static void RemoveCertificateFile()
{
}

private void InstanceMethod()
{
}
```
I can't remember if we formally agreed anything on ordering although I do have a vague memory of mentioning most visible -> least visible.

However, as it's a relatively small test class I'll leave it up to you whether you leave the ordering as it is now or tweak it.

Reply:
This is the default resharper ordering for a test class. Can we please add it tot he code standards list?